app/services/extractor.rb

```diff
@@ -0,0 +1,24 @@
+class Extractor < ApplicationService
+  def initialize(params, user)
+    @params = params
+    @user = user
+  end
+
+  def call; end
```

Comment:
This can be removed. This class now looks like an abstract one. If you do `Extractor.call(some_params, some_user)` in current implementation, it'll just silently do nothing. But if you remove this method (and couple below) it will then raise `NoMethodError` exception which is totally fine in this case.